tests/jerry/date-setters.js

```diff
@@ -82,25 +82,24 @@ assert (d.getUTCSeconds() == 1);
 assert (d.getUTCMilliseconds() == 1);
 
 /* 15.9.5.34 Date.prototype.setHours (hour [, min [, sec [, ms ] ] ] ) */
-// FIXME: Missing timezone adjustment.
-//d.setTime(0);
-//assert (d.setHours(1) == hour + d.getTimezoneOffset() * 60000);
-//assert (d.getHours() == 1);
-//d.setTime(0);
-//assert (d.setHours(1, 1) == hour + min + d.getTimezoneOffset() * 60000);
-//assert (d.getHours() == 1);
-//assert (d.getMinutes() == 1);
-//d.setTime(0);
-//assert (d.setHours(1, 1, 1) == hour + min + sec + d.getTimezoneOffset() * 60000);
-//assert (d.getHours() == 1);
-//assert (d.getMinutes() == 1);
-//assert (d.getSeconds() == 1);
-//d.setTime(0);
-//assert (d.setHours(1, 1, 1, 1) == hour + min + sec + ms + d.getTimezoneOffset() * 60000);
-//assert (d.getHours() == 1);
-//assert (d.getMinutes() == 1);
-//assert (d.getSeconds() == 1);
-//assert (d.getMilliseconds() == 1);
+d.setTime(0);
```

Comment:
There are several `FIXME:`s in [date-getters.js](https://github.com/jerryscript-project/jerryscript/blob/master/tests/jerry/date-getters.js#L28)  as well due to the timezone adjustment. Could you check them please?

Reply:
Done, and they pass (with some changes, the original tests were slightly incorrect).